### PR TITLE
Make table full width when mouse hovered

### PR DESCRIPTION
**Pull request by adesege**
### Why?
The Type column in dev doc tables is invisible without scrolling the column, even on a very wide browser because of an artificially narrow center div.

### What?
This PR makes the table in a page full width when mouse hovered. It also adds a background color to hide the table transparency when scrolled over another element/content.

### How?

1. Checkout to this branch locally.
1. In  cloudflare-docs repo, cd into `products/1.1.1.1` or any other products.
1. Point the `cloudflare-docs-engine` package to it's local version. Usually, `../../../cloudflare-docs-engine` if cloudflare-docs-engine and cloudflare-docs folders are in the same directory.
1. Install dependencies using `yarn install`.
1. Then run `yarn bootstrap` and `yarn start` to run the product in development environment.
1. Point your browser to http://localhost:8000/reference/log-fields/zone/spectrum_events and http://localhost:8000/logpull/requesting-logs

### CODE REVIEW

**Comment by adesege** [comment]
> Once this PR gets merged, I will regenerate the lock files of the products in cloudflare-docs so they can have the latest changes

File: src/css/docs/components/table-wrap.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@media (hover: hover) {
+  .DocsMarkdown--table-wrap:hover {
+    width: calc(100% + var(--docs-body-sidebar-width));
+  }
+
+  .DocsMarkdown--table-wrap:hover table {
+    background-color: #fff;
```
Comment (jrf0110) [inline src/css/docs/components/table-wrap.css:7]: Shouldn't this be

```css
background-color: var(--background-color);
```

And then we can remove the `[theme="dark"]...` declaration?

Comment (adesege) [reply src/css/docs/components/table-wrap.css:7]: Thanks. I will push a fix accordingly.

**Review by jrf0110 — APPROVED** [review]
LGTM but shouldn't we put these declarations in src/css/docs/components/docs-markdown.css since that's where the rest of the markdown table declarations are?

**Comment by adesege** [comment]
I will push a fix shortly

**Pull request merged**